common/lib/xmodule/xmodule/modulestore/store_utilities.py

```diff
@@ -3,6 +3,9 @@
 from collections import namedtuple
 
 import uuid
+from xblock.core import XBlock
+
+DETACHED_CATEGORIES = [name for name, __ in XBlock.load_tagged_classes("detached")]
```

Comment:
As I was looking through the documentation on opaque keys today I noticed this: https://opaque-keys.readthedocs.org/en/stable/opaque_keys.edx.html#opaque_keys.edx.locator.BlockUsageLocator.category

Since category is depreciated we may want to change the name of this to DETACHED_XBLOCK_TYPES